README.md

```diff
@@ -1,13 +1,17 @@
-# AWS IoT Device SDK C v4.0.0
+# AWS IoT Device SDK C v4_beta_deprecated
 
-**[Link to API documentation](https://docs.aws.amazon.com/freertos/latest/lib-ref/c-sdk/main/index.html)**
+**[Link to API documentation](https://docs.aws.amazon.com/freertos/latest/lib-ref/embedded-csdk/v4.0_beta_deprecated/lib-ref/index.html)**
 
 [![Build Status](https://travis-ci.org/aws/aws-iot-device-sdk-embedded-C.svg?branch=v4_beta_deprecated)](https://travis-ci.org/aws/aws-iot-device-sdk-embedded-C)
 [![codecov](https://codecov.io/gh/aws/aws-iot-device-sdk-embedded-C/branch/v4_beta_deprecated/graph/badge.svg)](https://codecov.io/gh/aws/aws-iot-device-sdk-embedded-C)
 
-The AWS IoT Device SDK for C is a collection of C99 source files that allow applications to securely connect to the AWS IoT platform. It includes an MQTT 3.1.1 client, as well as libraries specific to AWS IoT, such as Thing Shadows. It is distributed in source form and may be built into firmware along with application code.
+The CSDK v4_beta_deprecated branch contains a beta version of the AWS IoT Embedded C SDK libraries, which has now been deprecated. The libraries in this branch will not be released in its current form, and they are being refactored for resource constrained devices and go through code quality checks code quality checks including for [GNU Complexity](https://www.gnu.org/software/complexity/manual/complexity.html), [MISRA coding standard](https://www.misra.org.uk/MISRAHome/MISRAC2012/tabid/196/Default.aspx), [Coverity statical analysis](https://scan.coverity.com/), and [AWS CBMC automated reasoning tool](https://www.youtube.com/watch?v=YwQHAPRhQkI&feature=youtu.be&t=1721) to validate memory safety and functional correctness proof. Once the libraries have met the code quality checks, they will be released into the master branch.  
```

Comment:
go through code quality checks code quality checks
Please remove one 'code quality check'

Reply:
This paragraph could give a misleading message that the v4_beta_deprecated API would be re-factored and then released..

It will be important to mention that the refactored libraries will not be backwards compatible with the v4_beta_deprecated API.

Reply:
The use of "they are being refactored for.." phrase could be misgiving for libraries that are already refactored and part of the master branch. We should consider clearly mentioning that the MQTT and Shadow libraries are refactored and present in the master branch, while other libraries are undergoing refactor